Cargo.lock

```diff
@@ -1619,7 +1666,7 @@ dependencies = [
  "encoding_rs",
  "futures",
  "import_map",
- "indexmap",
+ "indexmap 2.3.0",
```

Comment:
@devsnek could you try to remove so many duplicate versions of various dependencies added in this PR? Maybe we could bump some of the version we use in Deno?